src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,14 @@
      * "Can't get min value from list: method_input_list"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        OptionalInt min = numbers.stream()
+                .map(s -> s.split(","))
+                .flatMap(a -> Arrays.stream(a))
+                .mapToInt(s -> Integer.parseInt(s))
+                .filter(i -> i % 2 == 0)
+                .min();
+        return min.orElseThrow(() -> new RuntimeException("Can't get min value from list:"
```

Comment:
You can add .orElseThrow right after min(). And you don`t need to create variable, write it right after return.
return numbers.stream()
```suggestion
                .min();
        return min.orElseThrow(() -> new RuntimeException("Can't get min value from list:"
```
                .map(s -> s.split(","))
                .flatMap(a -> Arrays.stream(a))
                .mapToInt(s -> Integer.parseInt(s))
                .filter(i -> i % 2 == 0)
                .min()
                .orElseThrow(() -> new RuntimeException("Message"))

Reply:
rewrited